tcms/telemetry/templates/telemetry/testcase-breakdown.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+{% load comments %}
+{% load extra_filters %}
+{% load attachments_tags %}
+
+{% block title %}TestCase Breakdown{% endblock %}
+
+{% block contents %}
+
+    <div class="col-md-3">
+        Product:
+    </div>
+    <div class="col-md-3">
+        <select name="product" id="id_select_product" class="form-control selectpicker">
+            {% for product in products %}
+                <option value="{{ product.pk }}">
+                    {{ product.name }}
+                </option>
+            {% endfor %}
+        </select>
+    </div>
+
+    <div class="col-md-3">
+        Test Plan:
+    </div>
+    <div class="col-md-3">
+        <select name="test_plan" id="id_select_test_plan" class="form-control selectpicker">
+            {% for test_plan in test_plans %}
+                <option value="{{ test_plan.pk }}">
+                    {{ test_plan.name }}
+                </option>
+            {% endfor %}
+        </select>
+    </div>
+
+    <div class="col-md-3" style="padding: 15px; margin: 15px; text-align: center">
+        <div style="text-align: center; font-weight: bold"> Automated/ Manual </div>
+        <div id="total"></div>
+        <div> TOTAL </div>
+
+        <div id="automated-bar" style="padding: 10px; font-weight: bold"></div>
+    </div>
+
+    <div class="col-md-4" style="padding: 15px; margin: 15px; text-align: center">
+        <div style="text-align: center; font-weight: bold"> Priorities </div>
+
+        <div id="priorities-chart"></div>
+        <svg></svg>
+    </div>
+
+    <div class="col-md-4" style="padding: 15px; margin: 15px; text-align: center">
+        <div style="text-align: center; font-weight: bold"> Categories </div>
+        <div id="categories-chart"></div>
+    </div>
+
+    <script src="{% static 'bootstrap-select/dist/js/bootstrap-select.min.js' %}"></script>
+    <script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.min.js' %}"></script>
+    <script src="{% static "grappelli/jquery/jquery.min.js" %}"></script>
+    <script src="{% static "grappelli/js/grappelli.min.js" %}"></script>
```

Comment:
why do you need grapelli here?

Reply:
For whatever reason, when I remove it I get `TypeError: $(...).selectpicker is not a function`

Reply:
You definitely don't need grappelli here. That problem has something to do with bootstrap-select not being loaded on time maybe.

The base template loads jquery in <head> so it should be defined. Grappelli has its own version of jQuery which you don't want to use.